edbdeploy/projects/virtualbox.py

```diff
@@ -378,32 +403,90 @@ def _build_virtualbox_ips(self, env):
                          "please check the logs for details.")
                         % env.cloud_spec['pooler_server_%s' % i]['name']
                     )
+        for i in range(env.cloud_spec['dbt2_client']['count']):
```

Comment:
Does it make sense here?  If it doesn't exists at this point, then I feel it should error out because the spec handling was correct prior to this.  Or is it better to be able to continue on?